docs/reference/images.md

```diff
@@ -336,7 +337,7 @@ $ sudo ig image inspect -o custom --extra-info=ebpf.sections ghcr.io/inspektor-g
 ["",".strtab",".text","kprobe/tcp_v4_connect",".relkprobe/tcp_v4_connect","kretprobe/tcp_v4_connect",".relkretprobe/tcp_v4_connect","kprobe/tcp_v6_connect",".relkprobe/tcp_v6_connect","kretprobe/tcp_v6_connect",".relkretprobe/tcp_v6_connect","kprobe/tcp_close",".relkprobe/tcp_close","kprobe/tcp_set_state",".relkprobe/tcp_set_state","kretprobe/inet_csk_accept",".relkretprobe/inet_csk_accept",".rodata",".bss","license",".maps",".BTF",".rel.BTF",".BTF.ext",".rel.BTF.ext",".llvm_addrsig",".symtab"]
 
 # Listing programs
-$ sudo ig image inspect -o custom fsnotify --extra-info ebpf.programs|jq -r '.[].Section'
+$ sudo ig image inspect -o custom fsnotify --extra-info ebpf.programs --jsonpath='[*].Section'
 WARN[0001] image signature verification is disabled due to using corresponding option 
 kprobe/fanotify_handle_event
 kprobe/fsnotify_remove_first_event
```

Comment:
The formatting in the docs does not match what I get:
```
$ sudo ig image inspect -o custom fsnotify --extra-info ebpf.programs --jsonpath='[*].Section'
["kretprobe/inotify_handle_inode_event","kprobe/fsnotify_insert_event","kretprobe/fsnotify_remove_first_event","kprobe/inotify_handle_event","kretprobe/inotify_handle_event","kprobe/fanotify_handle_event","kprobe/fsnotify_destroy_event","kprobe/inotify_handle_inode_event","kretprobe/fanotify_handle_event","kprobe/fsnotify_remove_first_event"]
```

I suggest to use `json.MarshalIndent` and then to make sure the output in the docs matches the real output.